.github/workflows/deploy-prod.yml

```diff
@@ -87,6 +87,11 @@ jobs:
           aws s3 cp public/list_integration_sdks.html s3://${{ secrets.AWS_PROD_S3_BUCKET_NAME }}/latest/modern/list_integration_sdks.html --cache-control max-age=3600
           AWS_MAX_ATTEMPTS=10 aws cloudfront create-invalidation --distribution-id ${{ secrets.AWS_PROD_CF_DISTRIBUTION_ID }} --paths "/latest*"
 
+      - name: Sync assets to S3
+        run: |
+          aws s3 cp assets/adobe-analytics-js/adobe-analytics-heartbeat.js s3://${{ secrets.AWS_PROD_S3_BUCKET_NAME }}/adobe-analytics-js/adobe-analytics-heartbeat.js --cache-control max-age=3600
+          aws s3 cp assets/adobe-analytics-js/adobe-analytics-js.js s3://${{ secrets.AWS_PROD_S3_BUCKET_NAME }}/adobe-analytics-js/adobe-analytics-js.js --cache-control max-age=3600
```

Comment:
Let's copy the whole assets directory instead of just `adobe-analytics`. It's safe to assume, we'll just place the files that we would want in S3.

Reply:
Also, let's rename and move the files from `adobe-analytics-js` to `integrations/AdobeAnalytics`. This will also match the integration's directory name.

Reply:
> Also, let's rename and move the files from adobe-analytics-js to integrations/AdobeAnalytics. This will also match the integration's directory name.

this requires changes in the integration, currently we don't want to do that.